framework/CodeInterfaces/Prescient/PrescientCodeInterface.py

```diff
@@ -179,7 +189,7 @@ def finalizeCodeOutput(self, command, codeLogFile, subDirectory):
       directory = subDirectory
     readFile = os.path.join(directory, toRead)
     if toRead.lower().startswith("hourly"):
-      busData, busList, busDataList = self._readBusData(os.path.join(directory, "bus_detail.csv"))
+      busData, busList, busDataList, hasMinute = self._readBusData(os.path.join(directory, "bus_detail.csv"))
```

Comment:
do we need to check here if hasMinute is available?

Reply:
No, hasMinute is the check.